Library/Homebrew/utils/github/api.rb

```diff
@@ -48,10 +48,10 @@ def initialize(github_message)
     class RateLimitExceededError < Error
       def initialize(reset, github_message)
         @github_message = github_message
+        new_pat_message = API.credentials ? "" : ", or:\n#{CREATE_GITHUB_PAT_MESSAGE}"
```

Comment:
Love the idea! How about this? I think it's a bit clearer.

```suggestion
        new_pat_message = ", or:\n#{CREATE_GITHUB_PAT_MESSAGE}" if API.credentials.blank?
```

Reply:
Good idea!